catroid/src/main/java/org/catrobat/catroid/formulaeditor/FormulaElement.java

```diff
@@ -819,7 +819,14 @@ private Object interpretObjectSensor(Sensors sensor, Sprite sprite) {
 				returnValue = (double) sprite.look.getTransparencyInUserInterfaceDimensionUnit();
 				break;
 			case OBJECT_LAYER:
-				returnValue = (double) sprite.look.getZIndex() - Constants.Z_INDEX_NUMBER_VIRTUAL_LAYERS;
+				if (sprite.look.getZIndex() < 0) {
+					returnValue = (double) ProjectManager.getInstance().getCurrentlyEditedScene().
+							getSpriteList().indexOf(sprite);
+				} else if (sprite.look.getZIndex() == 0) {
+					returnValue = 0d;
```

Comment:
Is this path tested?

Reply:
Nowadays I can check that myself, thanks to @mfuchs  https://jenkins.catrob.at/job/Catroid/job/PR-3065/1/cobertura/org_catrobat_catroid_formulaeditor/FormulaElement_java/ 

@mfuchs, does this report include all android tests (e.g. espresso, …)? (it looks like it but just to be sure :wink:)

Reply:
test is now in place, but i cant find the cobertura coverage report for run 15, @robertpainsi where is that hidden on jenkins?

Reply:
The coverage report isn't bound to the testrun but to the pull request, e.g. https://jenkins.catrob.at/job/Catroid/job/PR-3065, menu item [Coverage: instrumented_unit](https://jenkins.catrob.at/job/Catroid/job/PR-3065/Coverage_3a_20instrumented_5funit)

I'll ask if the Jenkins team can bind it to the testrun itself.

Reply:
Ignore my last comment. It is normally bound to the testrun itself but somehow got lost?